internal/pkg/auth/storage.go

```diff
@@ -136,7 +136,7 @@ func GetAuthField(key authFieldKey) (string, error) {
 		var errFallback error
 		value, errFallback = getAuthFieldFromEncodedTextFile(key)
 		if errFallback != nil {
-			return "", fmt.Errorf("write to keyring failed (%w), tried write to encoded text file: %w", err, errFallback)
+			return "", fmt.Errorf("read from keyring failed (%w), read from encoded file also failed: %w", err, errFallback)
```

Comment:
Thanks for trying to improve the error handling here, but was there any error message coming from here that was unclear?

The practise we are following for our Go error wrapping is that wrapper strings do not need to mention that something failed if that is already clear in the innermost error. E.g.:
- This: `send message: establish connection: connection refused`
- Instead of this: `sending message: establishing connection: connection refused`

Reply:
On a second though, this is kind of a special case with two inner errors and I agree with your suggested improvement :+1:

Reply:
We could also write
`fmt.Errorf("read from keyring: %w, read from encoded file as fallback: %w", err, errFallback)`

Reply:
I changed the error because pervously it said "write to keyring failed" but this is the `GetAuthField` function which reads and not writes :)

Reply:
Ah didn't even notice that, even better than. Thanks!